types/context.d.ts

```diff
@@ -0,0 +1 @@
+export interface FastifyContext {}
```

Comment:
Should there be to be some TS/JS Doc explaining this?

Reply:
Similarly to SchemaCompiler, the previous types had `any` here. I'm going to go ahead and use `unknown` as well as add a proper comment for now.

Reply:
Per this little bit of documentation it seems like I can use a `{ [key: string]: any }` here
https://github.com/fastify/fastify/blob/master/docs/Routes.md#config

Reply:
🤷‍♀️ this feature actually isn't as complicated as I thought. I'm going to fully implement the types!

Reply:
@fastify/core is `context` available on both `request` and `reply`?

Reply:
It's only on `reply`.